src/views/StackView/StackViewLayout.js

```diff
@@ -33,7 +33,10 @@ const IS_IPHONE_X =
   Platform.OS === 'ios' &&
   !Platform.isPad &&
   !Platform.isTVOS &&
-  (WINDOW_HEIGHT === 812 || WINDOW_WIDTH === 812);
+  (WINDOW_HEIGHT === 812 ||
```

Comment:
It'll be nice move these constants up and add comments to signal which devices they refer to, since they no longer refer to a single device:

```js
IPHONE_X_HEIGHT = 812; // iPhone X and XS
IPHONE_XR_HEIGHT = 896; // iPhone XR and XS Max
```

Reply:
Great idea, it will avoid confusion in the future why there are two values. I just pushed a new commit with those two constants and comments. Thank you!